docs/rst/fastdds_qos_profiles_manager_cli/cli_verbs/cli_set_verb.rst

```diff
@@ -2,3 +2,68 @@
 
 Fast DDS QoS Profiles Manager CLI ``set`` verb
 ----------------------------------------------
+
+Passing to the CLI the verb ``set`` as the third argument would include the given ``value`` in the selected XML ``element``.
```

Comment:
I suggest giving the information split for better consumption hehehe

```suggestion
Passing to the CLI the verb ``set`` would include the given ``value`` in the selected XML ``element``.
The verb is the third argument passed to the CLI (reference to the usage section).
The element is given by a dot-separated string accessing the configuration parameter to be set (reference to the Configuration parameter access section).
The set command requires usually at least the value desired for the specified element.
(Here the note stating the exception to the one value can be)
(Once complex commands are supported, this section will need to be revisited).
```

Reply:
Notes were taken to revisit this section when complex commands supported